test/test.repo.js

```diff
@@ -425,6 +426,25 @@ describe('Creating new Github.Repository', function() {
       });
    });
 
+   it('should merge pull requests on repo', function(done) {
+      var repo = github.getRepo('michael', 'github');
+      var expectedDocUrl = 'https://developer.github.com/v3/pulls/#merge-a-pull-request-merge-button';
+
+      repo.getPull(153, function(err, pull) {
+         repo.mergePull(pull, function(err) {
```

Comment:
I agree, but given there's no mocking of the github API, I don't see a way to do merges in a repeatable, parallel-safe, way.
But maybe the test title could be negative then.